recipes-multimedia/gstreamer/gstreamer1.0-plugins-base_1.22.5.imx.bb

```diff
@@ -125,12 +125,6 @@ S = "${WORKDIR}/git"
 
 inherit use-imx-headers
 
-# Prior to version 1.22.5, viv-fb is coupled with the Vivante direct texture feature.
-# For this reason, in these older versions, viv-fb must be enabled always, even when
-# building for SoCs like the i.MX8 that do not support the viv-fb windowing system.
-# TODO: Once this .imx recipe is upgraded to 1.22.5 or newer, drop this line.
-PACKAGECONFIG_GL:append = " viv-fb "
```

Comment:
meta-freescale also still has it for imx6: https://github.com/Freescale/meta-freescale/blob/master/recipes-multimedia/gstreamer/gstreamer1.0-plugins-base_1.22.%25.bbappend#L16
Does it make difference for imx7 since it doesn't have a GPU?

Reply:
I see, 7ULP does have GPU.